telebot/modules/admin.py

```diff
@@ -577,6 +578,56 @@ def pin(update: Update, context: CallbackContext):
     )
 
 
+@bot_action("enable")
+@check_user_admin
+def enable_command(update: Update, context: CallbackContext):
+    """
+    Enable commands for given chat
+    :param update: object representing the incoming update.
+    :param context: object containing data about the command call.
+    """
+    # check if any commands were passed
+    if not context.args:
+        update.effective_message.reply_text("I'm a cat, not a psychic! Mention the commands you want to enable...")
+        return
+
+    # enable commands in db
+    responses = enable_commands_for_chat(update.effective_chat.id, context.args)
```

Comment:
```suggestion
    if responses := enable_commands_for_chat(update.effective_chat.id, context.args):
        if len(responses) > 1:
             reply_markdown(blah)
        else:
             reply_markdown(responses[0])
        else:
            raise Exception(
```
Can keep it a bit cleaner like this